go.mod

```diff
@@ -55,6 +56,7 @@ require (
 	github.com/jaypipes/ghw v0.10.0
 	github.com/jedib0t/go-pretty/v6 v6.4.6
 	github.com/jhump/protoreflect v1.14.1
+	github.com/juju/ratelimit v1.0.2
```

Comment:
this module is LGPL, but https://pkg.go.dev/golang.org/x/time/rate appears to have the same functionality (and it's an existing dependency too)